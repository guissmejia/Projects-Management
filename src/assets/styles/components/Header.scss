.Header {
  width: 100%;
  height: auto;
  background: #fff;
  display: grid;
  grid-template-columns: 48% 48%;
  justify-content: center;
  &__logo {
    display: flex;
    figure {
      margin: 0;
    }
    img {
      width: 20%;
      position: relative;
    }
    h1 {
      position: absolute;
      left: 7%;
      font-weight: 500;
    }
  }
  &__user {
    display: flex;
    justify-content: flex-end;
    align-items: center;
    figure {
      margin: 0 .5em;
      background: #f9f9f9;
      border-radius: 50%;
      width: 10%;
      height: 65%;
      display: flex;
      justify-content: center;
      align-items: center;
    }
    img {
      width: 75%;
    }
    h2 {
      color: gray;
      font-weight: 400;
      font-size: 20px;
    }
  }
}
