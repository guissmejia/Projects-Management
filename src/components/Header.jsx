import React from "react";
import Logo from "../assets/static/figure.png";
import userIcon from "../assets/static/user.png";
import "../assets/styles/components/Header.scss";

const Header = () => {
  return (
    <header className="Header">
      <div className="Header__logo">
        <figure>
          <img 
            src={Logo} 
            alt="Projects Management Logo" 
          />
        </figure>
        <h1>Projects Management</h1>
      </div>
      <div className="Header__user">
        <figure>
          <img 
            src={userIcon} 
            alt="Header User" 
          />
        </figure>
        <h2>Guiselle MejÃ­a</h2>
      </div>
    </header>
  );
};

export default Header;
