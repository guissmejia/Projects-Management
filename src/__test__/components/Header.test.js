import React from 'react';
import Header from '../../components/Header';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';

describe('<Header/>', () => {
  test('Render header component', () => {
    const header = mount(<Header/>);
    expect(header.length).toEqual(1); 
  });
    test('Header Snapshot', () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
  });
});