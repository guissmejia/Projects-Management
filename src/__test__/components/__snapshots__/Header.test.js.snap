// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> Header Snapshot 1`] = `
<header
  className="Header"
>
  <div
    className="Header__logo"
  >
    <figure>
      <img
        alt="Projects Management Logo"
        src="test-file-stub"
      />
    </figure>
    <h1>
      Projects Management
    </h1>
  </div>
  <div
    className="Header__user"
  >
    <figure>
      <img
        alt="Header User"
        src="test-file-stub"
      />
    </figure>
    <h2>
      Guiselle Mej√≠a
    </h2>
  </div>
</header>
`;
